### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by sunng87**
This patch updates README to include some more information

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "db2",
```
Comment (sunng87) [inline package.json:2]: @sunchanglong do we need to update this name?

Comment (sunchanglong) [reply package.json:2]: 我不清楚了，要不直接改成 info8fcc-greptimedb-datasource ？

File: README.md [hunk 2]
```diff
@@ -1,40 +1,75 @@
...
+additional features.
 
-## Install packaged plugin Locally(Use of either method)
+## Screenshots
 
-> We haven't published this plugin to grafana, you can install local currently.
+PromQL query builder with additional field selector.
 
-> Get plugin zip from  [release page](https://github.com/GreptimeTeam/greptimedb-grafana-datasource/releases) 
+![explore](https://raw.githubusercontent.com/GreptimeTeam/greptimedb-grafana-datasource/doc/README/screenshots/1.png)
```
Comment (sunng87) [inline README.md:17]: FIXME: update this url when review finished.

File: README.md [hunk 3]
```diff
@@ -1,40 +1,75 @@
...
 
-1. ### Use docker compose (Recommend, contains dependency docker image)
-  * download [files in docker directory](https://github.com/GreptimeTeam/greptimedb-grafana-datasource/tree/main/docker)
-  * `cd` your downloaded directory, run `docker compose up` 
+Time-series data rendered with GreptimeDB data source.
+
+![dashboard](https://raw.githubusercontent.com/GreptimeTeam/greptimedb-grafana-datasource/doc/README/screenshots/2.png)
```
Comment (sunng87) [inline README.md:21]: FIXME: update this url when review finished.

**Review by sunchanglong — APPROVED** [review]

**Pull request merged**